globals.d.ts

```diff
@@ -16,4 +16,10 @@ declare global {
             writable: WritableStream<RTCEncodedAudioFrame | RTCEncodedVideoFrame>;
         }
     }
+    interface MediaStreamTrack {
```

Comment:
Won't these clash with the standard one? I thought we need to extend the default ones?